### ISSUE TITLE
nice graphics, would be nice to have them included in the repo

### ISSUE THREAD

**Issue by timmwille** [ceremcem/freecad-notes#11]
last question for now, as I sometimes work offline with git, it is nice to have the images/gifs included in a folder inside the repo, would this be possible? (I'm not sure where the sources are, the URLs looked a bit weird to me, but worst case one could just extract those one by one)

cheers

**Comment by ceremcem** [comment]
I don't have *all* the images, gifs and videos in my possession (existing ones are in temporary folders and not ordered) because I rely on having internet and servers. However, if offline working is a necessity, then it's a necessity. 

I'll add any new media into the repo. Current media might be progressively transferred here.

**Comment by ceremcem** [comment]
Idea: Maybe all tutorials should reside in their own directory in this repo, including its media and project files, etc. Then I may write a compiler that generates the `repo/README.md` in current table format.

**Comment by timmwille** [comment]
Sounds great! I'm checking this repo regularly now, so let me know if I shall have a specific look at something :)

**Comment by ceremcem** [comment]
Please do not expect any huge progress in near future because currently I'm busy with non-mechanical tasks and these HOWTO's should be reviewed again during the refactoring, because the Asm3 WB has been evolved during this period. 

I might cleanup the "Assembly Dependent Design" HOWTO, as it's the latest one I used, but any help is appreciated. 

As for the checking the repo periodically: I'll never push the changes to master branch directly but I'll create a PR instead, so everyone watching the repository will be notified when something is about to change.

**Comment by ceremcem** [comment]
There is a problem with "supporting offline view". The markdown which produces great results for offline viewing does not work for online viewing. See [here](https://github.com/ceremcem/freecad-notes/blob/d8731ac67dcab4ed37379fa6ea54f24682a3c9ee/tasks/assembly-dependent-design/README.md).

If you download the branch, you will see that the index.html looks fine and videos are shown within the page. However, README.md simply ignores the videos. 

![image](https://user-images.githubusercontent.com/6639874/106342131-c98a9200-62b0-11eb-8eb5-f3b7e6fb2abf.png)

Do you have any suggestions?

I have a few ideas: 

1. Completely disable Github preview by renaming `README.md` to something else and serve the documentation from some other location as HTML, like http://freecad-notes.surge.sh/
2. Somehow upload the videos to web, create a screenshot and use the `[![image](image-url)](video-url)` trick. This will introduce a synchronization headache as we need to keep local videos in sync with uploaded ones. Maybe a script would be written for this task.

**Comment by timmwille** [comment]
I would go with number 2. or have it like  it is now but add the URL and a short description underneath the Video as Option 3.

**Comment by timmwille** [comment]
I have not yet tested the Video embedding from Github much, but maybe there are more HTML Options, I can ask a friend

**Comment by timmwille** [comment]
https://www.nathan-lam.com/blog/embed-video-in-github-readme.html this might help?

**Comment by ceremcem** [comment]
Yes, that's the `[![image](image-url)](video-url)` trick. This trick requires uploading the videos before embedding the links into the Readme. We might upload the videos in static locations, again, like surge.sh or something.
